starknet/starknet-normalization/src/data.ts

```diff
@@ -0,0 +1,49 @@
+type FELT = string
+
+export interface BlockHeader {
+    hash: FELT
```

Comment:
FELT is the default byte array type in starknet.
In starknet-data, I have copied the structure from the specification, which highlights hashes, but still implies FELT like this
https://github.com/starkware-libs/starknet-specs/blob/b5c43955b1868b8e19af6d1736178e02ec84e678/api/starknet_api_openrpc.json#L1317
In starknet-normalization I simplified it because it doesn't need to be that specific imo.